src/main/java/com/github/webicitybrowser/spec/fetch/TaskDestination/TaskDestination.java

```diff
@@ -0,0 +1,10 @@
+package com.github.webicitybrowser.spec.fetch.TaskDestination;
+
+import com.github.webicitybrowser.spec.fetch.FetchConsumeBodyAction;
+
+
+public interface TaskDestination {
+
+	void enqueue(FetchConsumeBodyAction algorithm);
```

Comment:
This should take a Runnable, as it will be used for things other than just the consume body action